pkgs/tools/security/pinentry/mac.nix

```diff
@@ -32,6 +32,17 @@ stdenv.mkDerivation rec {
     chmod -R u+w macosx/*.nib
   '';
 
+  sandboxProfile = lib.optionalString stdenv.isDarwin ''
+    (allow process-exec
+      (literal "/usr/bin/ibtool")
+      (literal "/usr/libexec/PlistBuddy")
+      (regex "/Xcode.app/Contents/Developer/usr/bin/ibtool")
+      (regex "/Xcode.app/Contents/Developer/usr/bin/xcodebuild"))
```

Comment:
It would be unfortunate if Xcode were still necessary, as it negates the effort of using binary .nib files...

Reply:
I thought I pushed an update, but apparently it didn’t get pushed. Only PlistBuddy is required.

Reply:
It’s pushed. I did try with PlistBuddy from xcbuild, but it doesn’t accept multiple `-c` arguments.